src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -386,7 +419,12 @@ public void removeCommitWebHook(@NonNull BitbucketWebHook hook) throws IOExcepti
         if (StringUtils.isBlank(hook.getUuid())) {
             throw new BitbucketException("Hook UUID required");
         }
-        deleteRequest(V2_API_BASE_URL + owner + "/" + repositoryName + "/hooks/" + Util.rawEncode(hook.getUuid()));
+        String url = UriTemplate.fromTemplate(REPO_URL_TEMPLATE + "/hooks/{uuid}")
+                .set("owner", owner)
+                .set("repo", repositoryName)
+                .set("uuid", Util.rawEncode(hook.getUuid()))
```

Comment:
Don't think i need encoding here as Handy Uri should handle it

Reply:
Yep, remove the rawEncode as it’s `{uuid}`. You’d need to use raw encode if the template was `{+uuid}`